WordPress/UITestsFoundation/Screens/MySiteScreen.swift

```diff
@@ -258,6 +267,14 @@ public class MySiteScreen: ScreenObject {
         return self
     }
 
+    @discardableResult
+    public func verifyPagePublished(title: String) -> Self {
+        XCTAssertTrue(pagesCard.staticTexts[title].waitForExistence(timeout: 3))
+        XCTAssertTrue(pagesCardPublishedLabel.waitForExistence(timeout: 3))
```

Comment:
It's worth noting that the check for`Published` text presence is done globally within Pages Card, not for the exact page. This means, if there are two pages: one old which is published, and the newly created one, which failed to publish, the check will still pass.

IMO, it's not that critical, given that you're using a mock with one page. But it's worth remembering that this won't be as robust on a larger scale.

Reply:
thanks for pointing this out! yup this works for now because we're using a mocked response. yeah i also don't think it's critical to change at the moment as the test is expecting that mocked response but agree we might need to revisit if we change this test in the future.